example/index.res

```diff
@@ -1,3 +1,39 @@
+module ClickOutside = {
+  @react.component
+  let make = () => {
+    let inputRef = React.useRef(Js.Nullable.null)
+
+    let listRef = React.useRef(Js.Nullable.null)
+
+    let (isClickedOutside, setIsClickedOutside) = React.useState(_ => true)
```

Comment:
The wording is a bit confusing here, could it be `dropdownIsOpen` or `fruitsDropdownIsOpen`?